docs/workflow/building/coreclr/linux-instructions.md

```diff
@@ -1,108 +1,95 @@
-| Ubuntu 16.04  (x64, arm64 ROOTFS) | arm64 (arm64v8) | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-arm64-20210719121212-8a8d3be`        | `/crossrootfs/arm64` | -clang9       |
-| Alpine (x64, arm ROOTFS)    | arm             | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-arm-alpine-20210923140502-78f7860`   | `/crossrootfs/arm64` | -clang9       |
-| Alpine (x64, arm64 ROOTFS)  | arm64 (arm64v8) | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-arm64-alpine-20210719121212-b2c2436` | `/crossrootfs/arm64` | -clang9       |
+| OS                                | Target Arch     | Image location                                                                                       | crossrootfs location | Clang Version |
+| --------------------------------- | --------------- | ---------------------------------------------------------------------------------------------------- | -------------------- | ------------- |
+| Alpine                            | x64             | `mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-WithNode-20210910135845-c401c85`            | -                    | -clang5.0     |
+| CentOS 7 (build for RHEL 7)       | x64             | `mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-20210714125435-9b5bbc2`                        | -                    | -clang9       |
+| Ubuntu 18.04 (x64, arm ROOTFS)    | arm32 (armhf)   | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-20220426130400-6e40d49`              | `/crossrootfs/arm`   | -clang9       |
+| Ubuntu 18.04  (x64, arm64 ROOTFS) | arm64 (arm64v8) | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-arm64-20220427171722-6e40d49`        | `/crossrootfs/arm64` | -clang9       |
+| Alpine (x64, arm ROOTFS)          | arm             | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-arm-alpine-20210923140502-78f7860`   | `/crossrootfs/arm64` | -clang9       |
+| Alpine (x64, arm64 ROOTFS)        | arm64 (arm64v8) | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-arm64-alpine-20210923140502-78f7860` | `/crossrootfs/arm64` | -clang9       |
+| FreeBSD (x64 ROOTFS)              | x64             | `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-freebsd-12-20220831130538-f13d79e`   | `/crossrootfs/x64`   | (default)     |
 
-Environment
-===========
+Alternatively, for normal repo building you can use the latest (official releases and/or nightly) _SDK_ images found in their [official Microsoft Docker website](https://hub.docker.com/_/microsoft-dotnet-sdk/). Keep in mind that you might have to do additional setup for the _ROOTFS_ stuff if you want to do cross building for arm32/arm64 using those images. Instructions on how to do this can be found [here](/docs/workflow/building/coreclr/cross-building.md#linux-cross-building).
```

Comment:
If you start with the .NET SDK docker image, you would need to install all pre-requisites (clang, cmake, etc.) into it. It is same as if you start with plain vanilla Debian, or use no docker at all. 

Can you clarify why anybody would want to start with .NET SDK docker image to build the repo? If we give alternatives in the docs, we should explain why people may want to use the alternative.

Reply:
This is a good catch. It was some confusion on my part. I've been using the official .NET images to build lots of things as of recent, but not the repo. I will proceed to remove it.